packages/server-bridge/src/getConfigWithDefaults.js

```diff
@@ -2,10 +2,12 @@ import instrumentationMockFactory from './factories/instrumentationMockFactory';
 import stellarRequestFactory from './factories/stellarRequestFactory';
 
 export default function getConfigWithDefaults(config) {
+  const stellarRequest = stellarRequestFactory(config);
   return {
     instrumentation: instrumentationMockFactory(config),
-    stellarRequest: stellarRequestFactory(config),
+    stellarRequest,
     newSessionHandlers: [],
+    source: stellarRequest.source,
```

Comment:
source is already available in stellarRequest...

Reply:
but the people want it here!